blosc/blosc2.h

```diff
@@ -110,12 +110,14 @@ enum {
   BLOSC_BITSHUFFLE = 2,  //!< bit-wise shuffle
   BLOSC_DELTA = 3,       //!< delta filter
   BLOSC_TRUNC_PREC = 4,  //!< truncate precision filter
-  BLOSC_LAST_FILTER= 5,  //!< sentinel
+  BLOSC_UDFILTER = 5,  //!< user-defined filter
+  BLOSC_LAST_FILTER= 6,  //!< sentinel
 };
 
 enum {
   BLOSC2_MAX_FILTERS = 6,
   //!< Maximum number of filters in the filter pipeline
+  BLOSC2_MAX_UDFILTERS = 16,
```

Comment:
Add a comment here like:

// Maximum number of filters that a user can register.

Reply:
Fixed